src/javascript/binary/pages/trade/common.js

```diff
@@ -741,13 +741,14 @@ function marketOrder(market){
         france: 9,
         belgium: 10,
         amsterdam: 11,
-        commodities: 12,
-        metals: 13,
-        energy: 14,
-        volidx: 15,
-        random_index: 16,
-        random_daily: 17,
-        random_nightly: 18
+        otc_stock: 12,
```

Comment:
@binary-hamedanchi  the OTC_index, it is not in the list but it was sorted correctly after   americas: 7. Why ?